### PR TITLE
test(ui): Convert ProjectFilters test to RTL

### PR DESCRIPTION
**Pull request by malwilley**
Broke this test while making search improvements so RTL it is

### CODE REVIEW

File: tests/js/spec/views/projectDetail/projectFilters.spec.jsx [hunk 1]
```diff
@@ -13,46 +13,34 @@ describe('ProjectDetail > ProjectFilters', () => {
...
       <OrganizationContext.Provider value={organization}>
         <ProjectFilters query="" onSearch={onSearch} tagValueLoader={tagValueLoader} />
       </OrganizationContext.Provider>,
-      TestStubs.routerContext()
+      {context: TestStubs.routerContext(), organization}
     );
-    wrapper.find('SmartSearchBar textarea').simulate('click');
-    wrapper
-      .find('SmartSearchBar textarea')
-      .simulate('change', {target: {value: 'sentry.semv'}});
```
Comment (malwilley) [inline tests/js/spec/views/projectDetail/projectFilters.spec.jsx:37]: Not actually sure why the test used to type `sentry.semv` into the input before asserting that `release` is suggested. Assuming that was not intended but correct me if I'm wrong

File: tests/js/spec/views/projectDetail/projectFilters.spec.jsx [hunk 2]
```diff
@@ -13,46 +13,34 @@ describe('ProjectDetail > ProjectFilters', () => {
...
+        count: null,
+        firstSeen: null,
+        key: 'release.version',
+        lastSeen: null,
+        name: 'sentry@0.5.3',
+        value: 'sentry@0.5.3',
+      },
+    ]);
+    render(
       <OrganizationContext.Provider value={organization}>
```
Comment (scttcper) [inline tests/js/spec/views/projectDetail/projectFilters.spec.jsx:27]: You shouldn't need <OrganizationContext.Provider if you pass organization with the context

**Review by scttcper — APPROVED** [review]

File: tests/js/spec/views/projectDetail/projectFilters.spec.jsx [hunk 3]
```diff
@@ -13,46 +13,34 @@ describe('ProjectDetail > ProjectFilters', () => {
...
         <ProjectFilters query="" onSearch={onSearch} tagValueLoader={tagValueLoader} />
       </OrganizationContext.Provider>,
-      TestStubs.routerContext()
+      {context: TestStubs.routerContext(), organization}
```
Comment (priscilawebdev) [inline tests/js/spec/views/projectDetail/projectFilters.spec.jsx:30]: we discussed using the organization on this object or being explicit and wrapping the component with `<OrganizationContext.Provide` and decided that we would like to go with the second option. Maybe we should remove it from the object some time soon :)

**Review by priscilawebdev — APPROVED** [review]
lgtm

File: tests/js/spec/views/projectDetail/projectFilters.spec.jsx (hunk 3 above)
Comment (malwilley) [reply tests/js/spec/views/projectDetail/projectFilters.spec.jsx]: Sounds good!

**Pull request merged**